wpilib/wpilib/command/command.py

```diff
@@ -248,19 +256,19 @@ def isTimedOut(self):
             return (self.timeout != -1 and
                     self.timeSinceInitialized() >= self.timeout)
 
-    def getRequirements(self):
+    def getRequirements(self) -> Set['subsystem.Subsystem']:
         """Returns the requirements (as a set of Subsystems) of this command
         """
         with self.mutex:
             return self.requirements.copy()
 
-    def lockChanges(self):
+    def lockChanges(self) -> None:
         """Prevents further changes from being made
         """
         with self.mutex:
             self.locked = True
 
-    def setParent(self, parent):
+    def setParent(self, parent: 'commandgroup.CommandGroup') -> None:
```

Comment:
This type hint is absurd.

Reply:
That's how you have to do it with `TYPE_CHECKING`